csrankings.csv

```diff
@@ -13840,6 +13843,7 @@ Nalini Venkatasubramanian,Univ. of California - Irvine,https://www.ics.uci.edu/~
 Nalvo F. de Almeida Jr,UFMS,https://www.facom.ufms.br/~nalvo,kU9d-GIAAAAJ
 Nam Nguyen,Towson University,https://www.towson.edu/fcsm/departments/computerinfosci/facultystaff/nnguyen.html,aJcBWkgAAAAJ
 Nam Wook Kim,Boston College,https://www.namwkim.org,puTV6X0AAAAJ
+Namhoon Lee,UNIST,https://www.robots.ox.ac.uk/~namhoon/,wi9q5T8AAAAJ

```

Comment:
Please point to a website indicating this affiliation.

Reply:
Hello, 

It may take some time to update his own web page, as he's now on the annual leave for private vacation.

But, we'd like to point out that newly added data are for new faculty members joining the AI graduate school of UNIST.
You can find the information in the link below:
https://aigs.unist.ac.kr/eng/pages/sub02_01.php?Track=c120
Also, we'd like to point out that they are therefore capable of advising PhD students in CS (regardless of their home department.)

Could this be the evidence for your comment? 
If not, we'd like to reflect other requests first, except these two faculty members (Namhoon Lee and Gi-su Kim).
If necessary, we will make corresponding changes on our previous pull request.

Best regards,
Donguk

Reply:
Namhoon Lee has changed his webpage (http://www.robots.ox.ac.uk/~namhoon) including the UNIST affiliation. Also, you can see this page for more detail (https://aigs.unist.ac.kr/eng/pages/sub05_01_view.php?seq=1583&part=N4&rows=10&search_type=&search_word=&page=).